OutOfSchool/OutOfSchool.Common/Enums/Workshop/WorkshopType.cs

```diff
@@ -0,0 +1,9 @@
+using System.Text.Json.Serialization;
+
+namespace OutOfSchool.Common.Enums.Workshop;
+
+[JsonConverter(typeof(JsonStringEnumConverter))]
+public enum WorkshopType
+{
+    None
```

Comment:
Some TODO comment should be present here, otherwise this enum looks pretty simple